<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagesWizard</class>
 <widget class="QDialog" name="ImagesWizard">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Images</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>601</width>
       <height>361</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>581</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="localPathList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>581</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>300</y>
        <width>416</width>
        <height>60</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbAddImages">
         <property name="text">
          <string>Add images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemoveImage">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemoveAllImages">
         <property name="text">
          <string>Remove All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add images:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="pbNext">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
